packages/grid/x-data-grid-premium/src/DataGridPremium/DataGridPremium.tsx

```diff
@@ -823,8 +823,8 @@ DataGridPremiumRaw.propTypes = {
    */
   rowCount: PropTypes.number,
   /**
-   * If `single`, all column we are grouping by will be represented in the same grouping the same column.
-   * If `multiple`, each column we are grouping by will be represented in its own column.
+   * If `single`, all columns that are grouped by are represented in the same column.
+   * If `multiple`, each column that is grouped by is represented in its own column.
```

Comment:
I don't understand this. What does it mean for the columns to be grouped "by"? Grouped by what, exactly? Is there another way to say this? Should this be "grouped _together_" instead?